spec/controllers/request_controller_spec.rb

```diff
@@ -1042,6 +1042,69 @@ def expect_hidden(hidden_template)
     expect(response).to render_template('new_bad_contact')
   end
 
+  context "the outgoing message includes an email address" do
+
+    context "there is no logged in user" do
+
+      it "displays an error message warning not to include the email address" do
+        expected1 = "<p>You do not need to include your email in the " \
+                    "request in order to get a reply, as we will ask"
+        expected2 = "<p>We recommend that you edit your request and remove " \
+                    "the email address"
+
+        post :new, :info_request => {
+                     :public_body_id => @body.id,
+                     :title => "Test Request" },
+                   :outgoing_message => { :body => "me@here.com" },
+                   :submitted_new_request => 1,
+                   :preview => 1
+
+        expect(response.body).to include(expected1)
```

Comment:
I really wanted to check that the HTML wasn't being escaped as that was the error we had before from just passing the HTML into flash with the new cookie settings